Fix new hammer cli option added "ignore-global-proxy"

```
<tests.foreman.cli.test_hammer.HammerCommandsTestCase testMethod=test_positive_all_options>

>               '\n' + _format_commands_diff(self.differences)
            )
E           AssertionError: 
E           hammer repository create
E             Added options:
E               * ignore-global-proxy

```

This change is due to https://bugzilla.redhat.com/show_bug.cgi?id=1132980

--- a/tests/foreman/data/hammer_commands.json
+++ b/tests/foreman/data/hammer_commands.json
@@ -25890,6 +25890,12 @@
               "value": "GPG_KEY_ID"
             }, 
             {
+              "help": "if true, will ignore the globally configured Capsule when syncing.",
+              "name": "ignore-global-proxy",
+              "shortname": null,
+              "value": "IGNORE_GLOBAL_PROXY "
+            },
+            {
               "help": "", 
               "name": "label", 
               "shortname": null, 

